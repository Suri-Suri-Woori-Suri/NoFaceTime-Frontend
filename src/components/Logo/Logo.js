import React from 'react';

import styles from './Logo.module.css';
import logoImage from '../../assets/TwoFaceTime.gif';

const Logo = () => {
  return (
    <div className={styles.Logo}>
      <img
        src={logoImage}
        alt='Two Face Time gif'
        className={styles.LogoImage} />
    </div>
  );
};

export default Logo;
