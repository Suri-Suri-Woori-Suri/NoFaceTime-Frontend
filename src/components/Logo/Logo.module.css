.Logo {
  display: inline-flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: center;
  text-shadow: -5px 5px 5px grey;
  color: white;
  letter-spacing: 0px;
  font-family: "Anton", Arial, sans-serif;
  user-select: none;
  font-size: 45px;
  transition: all 0.05s ease-out;
  text-decoration: none;
  width: 70vw;
  height: 150px;
}

.LogoImage {
  height: 100%;
  margin-left: 30px;
}

.letter:hover {
  transform: translateY(1px);
  text-shadow: -4px 5px 4px grey;
}
