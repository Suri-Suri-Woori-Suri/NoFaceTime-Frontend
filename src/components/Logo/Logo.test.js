import React from 'react';
import { mount } from 'enzyme';
import { cleanup, render, screen } from '@testing-library/react';

import Logo from './Logo';

describe('<Logo />', () => {
  afterEach(cleanup);

  const wrapper = mount(<Logo />);

  it('should be contain Logo gif', () => {
    render(<Logo />);

    expect(screen.getByAltText('Two Face Time gif')).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
