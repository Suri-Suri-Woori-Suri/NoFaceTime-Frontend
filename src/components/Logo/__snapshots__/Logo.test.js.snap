// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> matches snapshot 1`] = `
<Logo>
  <div
    className="Logo"
  >
    <img
      alt="Two Face Time gif"
      className="LogoImage"
      src="TwoFaceTime.gif"
    />
  </div>
</Logo>
`;
